### PR TITLE
python3-shodan: update to 1.28.0.

### PR DESCRIPTION
**Pull request by HadetTheUndying**
#### Testing the changes
- I tested the changes in this PR: **briefly**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)

### CODE REVIEW

File: srcpkgs/python3-shodan/template [hunk 1]
```diff
@@ -13,7 +13,7 @@ maintainer="Alex Childs <misuchiru03+void@gmail.com>"
 license="MIT"
 homepage="https://developer.shodan.io"
 distfiles="https://github.com/achillean/shodan-python/archive/${version}.tar.gz"
-checksum=52932c6d508622877d0667e0290fee84e6cb749b6565b48ba359fbbf002f0845
+checksum=fc5f159f3edb6db0c2fff338b62f8a6c6eb63b12983bf729a4121fa30019753b
 make_check=extended
```
Comment (paper42) [inline srcpkgs/python3-shodan/template:17]: that should be =no I think

Comment (HadetTheUndying) [reply srcpkgs/python3-shodan/template:17]: On the `make_check`? Because GitHub highlighted the checksum line(16) for me.

Comment (paper42) [reply srcpkgs/python3-shodan/template:17]: It probably expects the comment next to $make_check or above it. @ericonr do you think `no` or `extended` makes more sense here? afaik we usually use extended for tests that take too long, not for tests that can't run without manually modifying the template to include a key.

**Comment by HadetTheUndying** [comment]
This is pretty much ready to go as soon as it's decided what to do with the `make_check` I've tested this pretty extensively this week

**Comment by paper42** [comment]
> This is pretty much ready to go as soon as it's decided what to do with the `make_check` I've tested this pretty extensively this week

+ you need to fix lint

**Comment by HadetTheUndying** [comment]
> > This is pretty much ready to go as soon as it's decided what to do with the `make_check` I've tested this pretty extensively this week
> 
> + you need to fix lint

As far as I know all that needs to be done there is explaining what the make_check is required for correct?

**Comment by paper42** [comment]
> > \+ you need to fix lint
> 
> As far as I know all that needs to be done there is explaining what the make_check is required for correct?

Just move the existing comment: https://github.com/void-linux/void-packages/pull/38112#discussion_r922892610

**Pull request merged**